### PR TITLE
External monitoring tools

### PR DESCRIPTION
**Pull request by Scitator**
Demo: https://app.wandb.ai/scitator/classification

### CODE REVIEW

File: catalyst/dl/__init__.py [hunk 1]
```diff
@@ -9,4 +9,4 @@
 from .experiment import BaseExperiment, SupervisedExperiment, ConfigExperiment
 from .meters import AverageValueMeter, ClassErrorMeter, ConfusionMeter, \
     MSEMeter, MovingAverageValueMeter, AUCMeter, APMeter, mAPMeter
-from .runner import SupervisedRunner
+from .runner import *
```
Comment (TezRomacH) [inline catalyst/dl/__init__.py:12]: we have to decide our code-style about `*`.

previously, we make an agreement to not use asterisks

File: catalyst/dl/runner/__init__.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
 # flake8: noqa
 
 from .supervised import SupervisedRunner
+from .wandb import *
```
Comment (TezRomacH) [inline catalyst/dl/runner/__init__.py]: also

Comment (TezRomacH) [reply catalyst/dl/runner/__init__.py]: maybe it's better to move this
```
try:
    import wandb
    WANDB_ENABLED = True
except ImportError:
    logger.warning(
        "wandb not available, switching to pickle. "
        "To install wandb, run `pip install wandb`."
    )
    WANDB_ENABLED = False

if WANDB_ENABLED:
    from .wandb import WandbRunner
```
here to simplify `catalyst/dl/runner/wandb.py` code?

File: catalyst/dl/runner/wandb.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+except ImportError:
+    logger.warning(
+        "wandb not available, switching to pickle. "
+        "To install wandb, run `pip install wandb`."
+    )
+    WANDB_ENABLED = False
+
+
+if WANDB_ENABLED:
+    class WandbRunner(Runner):
```
Comment (TezRomacH) [inline catalyst/dl/runner/wandb.py]: please, add some documentations

File: catalyst/utils/misc.py [hunk 4]
```diff
@@ -76,6 +76,17 @@ def append_dict(dict1, dict2):
     return dict1
 
 
+def flatten_dict(d, parent_key="", sep="/"):
```
Comment (TezRomacH) [inline catalyst/utils/misc.py:79]: is it better fit in Safitty?

**Review by TezRomacH — CHANGES_REQUESTED** [review]

File: catalyst/dl/__init__.py (hunk 1 above)
Comment (Scitator) [reply catalyst/dl/__init__.py:12]: It would be nice if you could propose workaround. For now - it looks like bad, but working solution.

Comment (TezRomacH) [reply catalyst/dl/__init__.py:12]: Personally, I prefer to write `__all__` in the script and then in init actually use `from .something import *`

Comment (TezRomacH) [reply catalyst/dl/__init__.py:12]: But i wonder why you demanded not to write `*` 🤔 in my previous PRs

**Pull request merged**